.vscode/launch.json

```diff
@@ -0,0 +1,13 @@
+{
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Debug",
+            "type": "gdb",
+            "request": "launch",
+            "target": "${workspaceFolder}/bazel-bin/<path to the bin file>",
```

Comment:
This is bazel specific and some some manual update for CMake?

Reply:
it can be used with CMake too, just need to change the target path.